osu.Game/Overlays/NewsOverlay.cs

```diff
@@ -7,81 +7,106 @@
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
 using osu.Framework.Graphics.Shapes;
-using osu.Game.Graphics;
+using osu.Game.Graphics.UserInterface;
 using osu.Game.Overlays.News;
+using osu.Game.Overlays.News.Displays;
 
 namespace osu.Game.Overlays
 {
     public class NewsOverlay : FullscreenOverlay
     {
+        private Container content;
+        private LoadingLayer loading;
         private NewsHeader header;
-
-        private Container<NewsContent> content;
-
-        public readonly Bindable<string> Current = new Bindable<string>(null);
+        private OverlayScrollContainer scrollFlow;
 
         public NewsOverlay()
             : base(OverlayColourScheme.Purple)
         {
         }
 
         [BackgroundDependencyLoader]
-        private void load(OsuColour colours)
+        private void load()
         {
             Children = new Drawable[]
             {
                 new Box
                 {
                     RelativeSizeAxes = Axes.Both,
-                    Colour = colours.PurpleDarkAlternative
+                    Colour = ColourProvider.Background5,
                 },
-                new OverlayScrollContainer
+                scrollFlow = new OverlayScrollContainer
                 {
                     RelativeSizeAxes = Axes.Both,
+                    ScrollbarVisible = false,
                     Child = new FillFlowContainer
                     {
                         RelativeSizeAxes = Axes.X,
                         AutoSizeAxes = Axes.Y,
                         Direction = FillDirection.Vertical,
                         Children = new Drawable[]
                         {
-                            header = new NewsHeader
-                            {
-                                ShowFrontPage = ShowFrontPage
-                            },
-                            content = new Container<NewsContent>
+                            header = new NewsHeader(),
+                            content = new Container
                             {
                                 RelativeSizeAxes = Axes.X,
                                 AutoSizeAxes = Axes.Y,
                             }
                         },
                     },
                 },
+                loading = new LoadingLayer(content),
             };
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            header.Post.BindValueChanged(onPostChanged, true);
```

Comment:
the fact that the *header* now controls the content display of its parent overlay is backwards. it should be the opposite (overlay controls content and reports changes to header, which is only for display).